module/zfs/zfs_onexit.c

```diff
@@ -107,30 +107,30 @@ zfs_onexit_destroy(zfs_onexit_t *zo)
  * of this function must call zfs_onexit_fd_rele() when they're finished
  * using the minor number.
  */
-int
+zfs_file_t *
 zfs_onexit_fd_hold(int fd, minor_t *minorp)
 {
 	zfs_onexit_t *zo = NULL;
-	int error;
 
-	error = zfsdev_getminor(fd, minorp);
+	zfs_file_t *fp = zfs_file_get(fd);
```

Comment:
There's a missing NULL check here, this appears to be what all the [CI bots hit](http://build.zfsonlinux.org/builders/Debian%2010%20x86_64%20%28TEST%29/builds/8762/steps/shell_4/logs/summary).

Reply:
Fixed. My tests were still running so glad the bots caught this!